packages/state-transition/src/block/processAttestationsAltairPlus.ts

```diff
@@ -152,7 +161,8 @@ export function getAttestationParticipationStatus(
 
   let flags = 0;
   if (isMatchingSource && inclusionDelay <= SLOTS_PER_EPOCH_SQRT) flags |= TIMELY_SOURCE;
-  if (isMatchingTarget && inclusionDelay <= SLOTS_PER_EPOCH) flags |= TIMELY_TARGET;
+  // post deneb attestation is valid till end of next epoch for target
+  if (isMatchingTarget && (fork >= ForkSeq.deneb || inclusionDelay <= SLOTS_PER_EPOCH)) flags |= TIMELY_TARGET;
```

Comment:
If forkSeq >= deneb and the attestation is more than the allowed end of next epoch old does this fall through as timely?

Reply:
attestation target is valid for this/previous epoch (see other validations) ... so not possible for inclusionDelay to be beyond next epoch (see other checks)